composer.json

```diff
@@ -15,13 +15,17 @@
         "php-http/message": "^1.6",
         "php-http/message-factory": "^1.0",
         "phpro/soap-client": "^0.7.11",
-        "psr/http-message": "^1.0"
+        "psr/http-message": "^1.0",
+        "symfony/property-access": "^3|^4",
+        "symfony/property-info": "^3.2|^4",
+        "symfony/serializer": "^3.2|^4"
```

Comment:
We should use 2 parallel pipe lines `||`

Reply:
omg... how the... what ?!

Reply:
One valid and good reason to user `composer-normalizer` from @localheinz :-)

Reply:
Fixed.